src/App.js

```diff
@@ -1,5 +1,130 @@
+const MissionUtils = require("@woowacourse/mission-utils");
+const Lotto = require("./Lotto");
+const LottoMachine = require("./LottoMachine");
+const Validation = require("./Validation");
+
 class App {
-  play() {}
-}
+  constructor() {
+    this.lottomachine = new LottoMachine();
+    this.purchaseAmount;
+    this.lottoQuantity;
+    this.lottoList;
+    this.winningList;
+    this.bonusNumber;
+    this.result = [0, 0, 0, 0, 0];
+    this.rateOfReturn;
+  }
+  play() {
+    this.getMoney();
+  }
+
+  getMoney() {
+    MissionUtils.Console.readLine("구입금액을 입력해 주세요.\n", (money) => {
+      this.purchaseAmount = parseInt(money);
+      const numberPattern = /^[0-9]*$/
```

Comment:
```suggestion
      const NUMBER_PATTERN = /^[0-9]*$/;
```
요런 식으로 변경해주시면 좋을 것 같긴합니다,